docs/src/trace-viewer-intro-js.md

```diff
@@ -55,19 +55,19 @@ The `trace-on` flag was introduced in Playwright v1.25. Check your `package.json
 
 ## Opening the HTML Report
 
-If you have a failed test then tests will run a total of 3 times. On the first retry the trace will be recorded. After the second retry the tests will stop running and a HTML report is available to view.
+The HTML report shows you a report of all your tests that have been ran and on which browsers as well as how long they took. Tests can be filtered by passed tests, failed, flakey or skipped tests. You can also search for a particular test. Clicking on a test will open the detailed view where you can see more information on your tests such as the errors, the test steps and the trace.
 
 ```bash
 npx playwright show-report
 ```
 
-In the HTML report click on the trace icon to directly open the trace file.
+In the HTML report click on the trace icon next to the test name file name to directly open the trace for the required test.
 
-<img width="960" alt="Playwright HTML Report" src="https://user-images.githubusercontent.com/13063165/189138532-bbb95411-3167-4f5f-985a-8886082fa7ab.png" />
+<img width="1404" alt="Playwright HTML Report" src="https://user-images.githubusercontent.com/13063165/212745273-c19487d2-bc5e-483f-9f67-f9c9e5413ff4.png" />
```

Comment:
There should be no incognito (3) at the top, I recommend to create a new Google Chrome profile for screenshots which looks like a normal Chrome.

Reply:
i dont understand what you are seeing. all my screenshots are taken with a google profile called playwright hence the green. this not only always gives me same colors but also same size as its always bumped up in size for better screenshots etc. where are you seeing incognito (3)??

Reply:
Never mind, seems like I meant the screenshots which were there before: 
![image](https://user-images.githubusercontent.com/17984549/212890225-0fa176f6-e358-4b72-9918-281250303b1a.png)

Your screenshots look good.